### DISCUSSION TITLE
原神正式入驻绳网啦！！！

### DISCUSSION THREAD

**Discussion by GenshinImpact01** [share121/inter-knot#157] [其他]
![ee84147e4bfe5f79000862dc715e5ed](https://github.com/user-attachments/assets/0a4f1d8e-5f2d-4c6a-ad2b-563e50a1d006)
《原神》是由米哈游自主研发的一款全新开放世界冒险游戏。游戏发生在一个被称作「提瓦特」的幻想世界，在这里，被神选中的人将被授予「神之眼」，导引元素之力。你将扮演一位名为「旅行者」的神秘角色，在自由的旅行中邂逅性格各异、能力独特的同伴们，和他们一起击败强敌，找回失散的亲人——同时，逐步发掘「原神」的真相。
（bushi）

**Comment by GenshinImpact01** [comment]
想你了卢迪克

**Comment by ghost** [comment]
启动!
